MySaver/Models/Product.cs

```diff
@@ -1,9 +1,41 @@
-﻿namespace MySaver.Models;
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
 
-public class Product //Should probably implement IEquatable for better comparison functionality
+namespace MySaver.Models;
+
+public class Product : INotifyPropertyChanged //Should probably implement IEquatable for better comparison functionality
 {
+    public event PropertyChangedEventHandler PropertyChanged;
+
+    private void OnPropertyChanged([CallerMemberName] String name = "")
+    {
+        PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(name));
+    }
+
     public string StoreName { get; set; }
+
     public string Name { get; set; }
-    public float Amount { get; set; } = 1;
-    public float Price { get; set; }
+
+    private float _amount = 1;
+    public float Amount
+    {
+        get { return _amount; }
+
+        set
+        {
+            _amount = value;
+            OnPropertyChanged();
+            OnPropertyChanged("Price");
+        }
+    }
+
+    public float UnitPrice { get; set; }
+
+    public float Price
+    {
+        get
+        {
+            return UnitPrice * Amount;
```

Comment:
Consider adding some sort of rounding here to show only two digits after comma.
![image](https://user-images.githubusercontent.com/113241847/201733336-1fdc00ea-35a0-488f-a31e-76241dc44ec6.png)

Reply:
Good catch! I thought simply formatting the result would be better than rounding, so that's what I did.